src/js/contentscripts/socialwidgets.js

```diff
@@ -219,6 +217,28 @@ function replaceButtonWithHtmlCodeAndUnblockTracker(button, tracker, html) {
   });
 }
 
+/**
+ * Unblocks the given tracker and replaces the given button with the widget
+ * element's HTML source code.
+ *
+ * @param {Element} button the DOM element of the button to replace
+ * @param {Tracker} tracker the Tracker object for the tracker that should be
+ *                          unblocked
+ * @param {HTMLElement} html the DOM element for the widget
```

Comment:
Do you mean `HTMLElement` to `widgetElement`? In that case, no, since `HTMLElement` is the actual type of what's being passed in. "widgetElement" is not a type.

Reply:
No, `html` -> `widgetElement` (the name of the variable)